python/cm/models.py

```diff
@@ -211,32 +221,65 @@ class ADCMEntity(ADCMModel):
     config = models.OneToOneField(ObjectConfig, on_delete=models.CASCADE, null=True)
     state = models.CharField(max_length=64, default='created')
     _multi_state = models.JSONField(default=dict, db_column='multi_state')
-    issue = models.JSONField(default=dict)
-    concern = models.ManyToManyField('ConcernItem', blank=True, related_name='%(class)s_entities')
+    _issues = models.ManyToManyField(
+        'ConcernItem', blank=True, db_column='issues', related_name='%(class)s_sources'
```

Comment:
Why we use db_column here, not just field name without _?

Reply:
underscore is used to hide it, of course. it's kind of internals not needed in API